lib/ustream.js

```diff
@@ -22,6 +24,12 @@ class Ustream {
    * @see authenticate for additional information about opts.
    */
   constructor (opts) {
+    API_RESOURCE_ENDPOINT = opts.endpoint || API_RESOURCE_ENDPOINT; // Customized endpoint, usefull for analytic API
```

Comment:
`API_RESOURCE_ENDPOINT` should remain a constant. Assign the endpoint to a class property instead.

```
this.endpoint = opts.endpoint || API_RESOURCE_ENDPOINT;
```

Reply:
Yes good but it implies updating the getResourceUrl function to use **this.endpoint** and no more the **API_RESOURCE_ENDPOINT**.